core/blockly.js

```diff
@@ -349,6 +349,20 @@ Blockly.hideChaff = function(opt_allowToolbox) {
 };
 
 /**
+ * When something in Blockly's workspace changes, call a function.
+ * @param {!Function} func Function to call.
+ * @return {!Array.<!Array>} Opaque data that can be passed to
+ *     removeChangeListener.
+ * @deprecated April 2015
+ */
+Blockly.addChangeListener = function(func) {
```

Comment:
Looks like scratch-blocks had removed this (since it's deprecated).  You probably don't want to add it back.